app/src/main/kotlin/com/wire/android/ui/home/HomeViewModel.kt

```diff
@@ -42,6 +44,17 @@ class HomeViewModel
             listenToEvents() // listen for the WebSockets updates and update DB accordingly
             loadUserAvatar()
         }
+        viewModelScope.launch {
+            ongoingCalls().collect {
+                if (it.isNotEmpty()) {
+                    navigationManager.navigate(
+                        command = NavigationCommand(
+                            destination = NavigationItem.IncomingCall.getRouteWithArgs(listOf(it.first().conversationId))
+                        )
+                    )
+                }
+            }
+        }
```

Comment:
For now we are getting the first occurrence of the incoming call, later on the ongoing calls usecase should also be changed to be able to receive only one call per time and not a list of incoming calls